sphinx/source/docs/user_guide/server.rst

```diff
@@ -671,74 +665,72 @@ locked session callback on a different update rate.
     doc.add_periodic_callback(update, 200)
     doc.add_root(p)
 
-As before, you can run this example by saving to a python file and running
+As before, you can run this example by saving to a Python file and running
 ``bokeh serve`` on it.
 
 .. _userguide_server_applications_hooks:
 
-Lifecycle Hooks
+Lifecycle hooks
 ~~~~~~~~~~~~~~~
 
-Sometimes it is desirable to have code execute at specific times in a server
-or session lifetime. For instance, if you are using a Bokeh Server alongside
-a Django server, you would need to call ``django.setup()`` once, as each
-Bokeh server starts, to initialize Django properly for use by Bokeh
-application code.
+You may want to execute code at specific points of server or session runtime.
+For instance, if you are using a Bokeh server with a Django server, you need to
+call ``django.setup()`` for each Bokeh server to properly initialize Django for
+use by Bokeh application code.
 
-Bokeh provides this capability through a set of *Lifecycle Hooks*. To use
-these hooks, you must create your application in
-:ref:`userguide_server_applications_directory`, and include a designated file
-called ``app_hooks.py`` in the directory. In this file you can include
-any or all of the following conventionally named functions:
+Bokeh enables this through a set of *lifecycle hooks*. To use these hooks,
+create your application in
+:ref:`directory format<userguide_server_applications_directory>` and include a
+designated file called ``app_hooks.py`` in the directory. In this file you can
+include any or all of the following conventionally named functions:
 
 .. code-block:: python
 
     def on_server_loaded(server_context):
-        ''' If present, this function is called when the server first starts. '''
+        '''If present, this function executes when the server starts.'''
         pass
 
     def on_server_unloaded(server_context):
-        ''' If present, this function is called when the server shuts down. '''
+        ''' If present, this function executes when the server shuts down. '''
         pass
 
     def on_session_created(session_context):
-        ''' If present, this function is called when a session is created. '''
+        ''' If present, this function executes when you create a session. '''
         pass
 
     def on_session_destroyed(session_context):
-        ''' If present, this function is called when a session is closed. '''
+        ''' If present, this function executes when you close a session. '''
```

Comment:
I don't think "you" is appropriate in this context.  Is this "you" the app author? Or the app viewer? Ultimately, it is the Bokeh server that creates and closes sessions, and I think it would be most unambiguous more like "when the server closes a session" (and creates, above)